### PR TITLE
update QE easyblock for version 6.5

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyblocks/q/quantumespresso.py [hunk 1]
```diff
@@ -415,11 +418,12 @@ def sanity_check_step(self):
...
         if 'pw' in targets or 'all' in targets:
             bins.extend(["dist.x", "ev.x", "kpoints.x", "pw.x", "pwi2xsf.x"])
-            if LooseVersion(self.version) >= LooseVersion("5.1"):
-                bins.extend(["generate_rVV10_kernel_table.x"])
-            if LooseVersion(self.version) > LooseVersion("5"):
-                bins.extend(["generate_vdW_kernel_table.x"])
-            else:
+            if LooseVersion(self.version) < LooseVersion("6.5"):
+                if LooseVersion(self.version) >= LooseVersion("5.1"):
+                    bins.extend(["generate_rVV10_kernel_table.x"])
```
Comment (akesandgren) [inline easybuild/easyblocks/q/quantumespresso.py]: Hmm, doesn't 6.5 build the generators any longer?
So where does it take the tables from then??

Comment (migueldiascosta) [reply easybuild/easyblocks/q/quantumespresso.py]: it generates them on-the-fly, they say it's fast (https://github.com/QEF/q-e/releases/tag/qe-6.5)

Comment (akesandgren) [reply easybuild/easyblocks/q/quantumespresso.py]: Ok, Then I bet that they are using the code that can't stand opt levels above -O0 and can barely produce correct results even then....

I sort of promised them to fix the vdw generator several years ago but never got around to it...

**Comment by akesandgren** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**